### ISSUE TITLE
mdToast announced twice by screen reader

### ISSUE THREAD

**Issue by EamonKavanagh** [angular/material#10853]
**Actual Behavior**:
 -  When an mdToast dialog pops up, it's announced twice by the screen reader. The transcript is:

[text in mdToast]
Alert
[text in mdToast]
Alert

 -  The text of the toast and alert only announced once.

**CodePen** (or steps to reproduce the issue): *
- See the demo on the Angular Material page here: https://material.angularjs.org/latest/demo/toast
- Tested using ChromeVox 53.0.2784.5 and we've had reports of this using JAWS as well.

**AngularJS Versions**: *
1.6.4-local+sha.617b36117

**Additional Information**:
- Chrome 59.0.3071.115

**Comment by Splaktar** [comment]
I am only seeing the Toast announced once with Version 68.0.3440.87 (Official Build) beta (64-bit). However, I am hearing "Remove" as the first thing when a toast is displayed.

**Comment by Splaktar** [comment]
This looks like a Chromevox bug so I've opened https://bugs.chromium.org/p/chromium/issues/detail?id=873324 to track this issue.

**Comment by Splaktar** [comment]
Latest update from the Chrome team was to remove the use of `aria-relevant` or to change it from `aria-relevant="all"` to `aria-relevant="additions text"`. However, doing this on macOS causes VoiceOver to not announce the toast after the first time it is opened. On Chromevox, it's worse, this causes toasts to not be announced at all.

I've provided feedback to them and requested that they reopen the bug.
